installer/targets/amigo-user/setup

```diff
@@ -230,6 +226,5 @@ alias amigo-copy-my-id="ssh-copy-id amigo@amigo1; ssh-copy-id amigo@amigo2"
 
 alias amigo-presentation-timer-start='rosrun challenge_final time.py __ns:=amigo'
 
-alias amigo-mount-usb='rosrun challenge_manipulation mount_usb'
+alias amigo-mount-usb='rosrun challenge_storing_groceries mount_usb'
```

Comment:
Lets move this executable outside the challenges. Having it here does not make sense